mcp/diskmanager/diskmanager_tng.c

```diff
@@ -476,7 +528,7 @@ static diskentry_t *diskpool_find_new_disk() {
 
 
 /**
- * Mounts a block device and returns the newly created mount point.
+ * @brief Mounts a block device and returns the newly created mount point.
  *
  * Loops through the following possible disk formats before exiting with failure:
  * ext3, ext4, xfs, jfs, reiser
```

Comment:
This no longer appears to be true. I see only xfs and ext4

Reply:
Fixed, thanks I didn't catch that char array got modified